cluster/etcd/stack.yaml

```diff
@@ -51,11 +51,6 @@ Resources:
           Fn::Base64: !Sub |
             #cloud-config
             write_files:
-              - path: /etc/scalyr-agent-2/userdata.yaml
-                permissions: 0644
-                content: |
-                  scalyr_api_key: "{{.Values.etcd_scalyr_key}}"
-                  stack_name: ${AWS::StackName}
```

Comment:
I think we should delete this only after we switch to the new etcd AMI. Then we can do an additional PR to delete.

Reply:
But the file path will be overwritten by the one defined in files.yaml (or it will fail if the path already exists, not sure).

In either case I don't see the benefit of keeping it, and requiring an additional PR?

Reply:
I will update this PR to include the AMI change so it's done atomically.

Reply:
You're right, it's the same file :facepalm: